os-server/src/main/java/app/openschool/discussion/DiscussionAnswerController.java

```diff
@@ -0,0 +1,61 @@
+package app.openschool.discussion;
```

Comment:
please move the discussion folder inside course, create two packages one for peers and one for mentors and separate classes with each other